content/tutorials/get-started/get-started-with-python-sdk.md

```diff
@@ -0,0 +1,202 @@
+
+**Warning:**  Never use publicly available servers to sign transactions. For more information about security and signing, see [](xref: set-up-secure-signing.md).
+
+**Caution:** Ripple provides the [Testnet and Devnet](parallel-networks.html) for testing purposes only, and sometimes resets the state of these test networks along with all balances. As a precaution, Ripple recommends **not** using the same addresses on Testnet/Devnet and Mainnet.
+
+
+If you only want to explore the XRP Ledger, you can  use the [Ledger Explorer](https://livenet.xrpl.org/) to see the Ledger progress in real-time and dig into specific accounts or transactions. 
+
+
+```py
+# Define the URL of the server you want to use
+JSON_RPC_URL = "http://s1.ripple.com:51234/"
+```
+
+### {{n.next()}}. Query
+
+Before you submit a transaction to the XRP Ledger, you should query it to check your account status and balances to make sure that the transaction will succeed. 
+
+
+```py
+# Look up information about your account
+def getAcctInfo():
+    client = JsonRpcClient(JSON_RPC_URL)
+    acct_info = AccountInfo(
+        account=address,
+        ledger_index="current",
+        queue=True,
+        strict=True,
+    )
+    response = client.request(acct_info)
+    print("response.status: ", response.status)
+    print("response.result: ", response.result)
+    print("response.id: ", response.id)
+    print("response.type: ", response.type)
+```
+
+
+### {{n.next()}}. Submit transaction
+
+Submitting a transaction to the XRP inolves three distinct steps:
+
+* Preparing the transaction.
+* Signing the transaction.
+* Submitting the transaction to an XRP Ledger node. 
+
+With `xrpl-py`, you can combine the these steps by using the `tx.to_dict` function and the `Sign` model. 
+
+
+```python
+ # Prepare the tx by formatting it into
+# the shape expected by the XRP Ledger
+# and signing it
+def prepareSignSubmitTx():
+    CLASSIC_ACCOUNT.next_sequence_num = get_next_valid_seq_number(CLASSIC_ACCOUNT, JSON_RPC_CLIENT)
+    tx = Transaction(
+        account=_ACCOUNT,
+        fee=_FEE,
+        sequence=CLASSIC_ACCOUNT.next_seuqnece_num + 10,
+        transaction_type=TransactionType.
+    )
+    value = tx.to_dict()["payment_trannsaction"]
+    signed_tx = Sign(
+        transaction=value,
+        seed=_SEED,
+        seed_hex=_SEED_HEX,
+    )
+    print("Signed transaction: ", signed_tx)
+    payment_transaction = Payment.from_dict(value)
+    response = send_reliable_submission(
+        payment_transaction, CLASSIC_ACCOUNT, JSON_RPC_URL
+        )
+    print("response.result.validated: ", response["validated"])
+    print("response.result.meta: ", response.result["meta"]["TransactionResult"], "tesSUCCESS")
+```
+
+
+### {{n.next()}}. Verify results
+
+To ensure that your transaction has succeeded in a validated ledger, which means that the results are final, you can add simple logic to your submit function:
```

Comment:
You may want to point out the difference between using a standard `submit` vs the `send_reliable_submission` above - without that specific method that does the heavy lifting, it doesn't double check validation.